src/FileServiceManager/FileService/DAL/Storages/FileSystemStorage/FileSystemStorage.cs

```diff
@@ -0,0 +1,132 @@
+﻿using FileService.Models;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.IO;
+using Microsoft.Extensions.Configuration;
+using FileService.Helpers;
+
+namespace FileService.DAL
+{
+    public class FileSystemStorage : IStorage
+    {
+        private readonly string _basePath;
+        private readonly int _fileSizeLimit;
+        private readonly IEnumerable<string> _allowedExtensions;
+        private readonly IFileSystemConfigurator _fileSystemConfigurator;
+
+        public FileSystemStorage(IConfiguration configuration, IFileSystemConfigurator fileSystemConfigurator)
```

Comment:
same here, use IOptions, do not pass IConfiguration here

Reply:
Changed